crates/wash-cli/tests/common/mod.rs

```diff
@@ -601,12 +601,12 @@ impl TestWashInstance {
             .context("failed to parse output of `wash app deploy`")
     }
 
-    /// Trigger the equivalent of `wash app list` on a [`TestWashInstance`]
-    pub(crate) async fn list_apps(&self) -> Result<AppListCommandOutput> {
+    /// Trigger the equivalent of `wash app get` on a [`TestWashInstance`]
+    pub(crate) async fn list_apps(&self) -> Result<AppGetCommandOutput> {
```

Comment:
Could we duplicate this test until we formally remove `wash app list` ?

Reply:
Yes, I was thinking about as well. Re-added the two different version and tested the output of those. 🙇